docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -31,32 +31,22 @@ An example deployment scenario:
 == Deploying your first devfile
 
 [discrete]
-===== Prerequisites
 
-* Before proceeding, you must know your ingress domain cluster name. For example: `apps-crc.testing` is the cluster domain name for https://github.com/code-ready/crc[Red Hat CodeReady Containers]
+==== Prerequisites for Openshift Cluster
+
 * Enable experimental mode for odo. This can be done by: `odo preference set experimental true`
 
-== Creating a project
+* Log in to a OpenShift cluster: `odo login -u developer -p developer`
 
-Create a project to keep your source code, tests, and libraries
-organized in a separate single unit.
+* Create a project: `odo project create myproject`
 
-. Log in to a OpenShift cluster:
-+
-[source,sh]
-----
-  $ odo login -u developer -p developer
-----
 
-. Create a project:
-+
-[source,sh]
-----
-  $ odo project create myproject
-   ✓  Project 'myproject' is ready for use
-   ✓  New project created and now using project : myproject
-----
-+
+==== Prerequisites for Kubernetes Cluster
+
+* Enable experimental mode for odo. This can be done by: `odo preference set experimental true`
+
+* Before proceeding, you must know your cluster domain name to specify host for ingress.
```

Comment:
We should provide link where users can learn how to do that, or what the ingress host is.

Reply:
I also think so, tried an exact way to determine cluster domain name, what i understand till now it is much dependent on the way cluster is deployed.
I have also seen this issue to add document details on how to get `--host` for url https://github.com/openshift/odo/issues/3463